sycamore/scans/file_scan.py

```diff
@@ -129,13 +133,24 @@ def _to_document(self, dict: dict[str, Any]) -> dict[str, bytes]:
             dict["path"] = "s3://" + dict["path"]
         properties.update({"path": dict["path"]})
         if "filetype" not in properties and self._binary_format is not None:
-            properties["filetype"] = self._binary_format
+            properties["filetype"] = self._file_mime_type()
         if self._metadata_provider:
             properties.update(self._metadata_provider.get_metadata(dict["path"]))
         document.properties = properties
 
         return {"doc": document.serialize()}
 
+    def _file_mime_type(self):
+        if self._binary_format == "html" or self._binary_format == "htm":
+            return "text/html"
+        if self._binary_format == "pdf":
+            return "application/pdf"
+        if self._binary_format == "pptx":
+            return "application/vnd.openxmlformats-officedocument.presentationml.presentation"
+        ret = f"application/{self._binary_format}"
+        logger.warning(f"Unrecognized extenstion {self._binary_format}; using {ret}")
+        return ret
+
```

Comment:
It's unfortunate that we have to reinvent this wheel.  Can we use Python's mimetypes.guess_type()?  We should at least add `.ppt` and both `.doc` and `.docx`. I've seen `.doc` in the sort benchmark crawl.  It would be good to just make this table-driven.  Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types I'd suggest also adding csv, tsv (text/tab-separated-values), rtf, xls, xlsx, and xml.  There's reasonable likelihood we could ingest those.

Reply:
I changed it to support mimetypes.guess_type. 
I'd previously only handled stuff we partitioned.